code/obj/item/organ_holder.dm

```diff
@@ -826,6 +826,14 @@
 						H.wear_mask = newHead.wear_mask
 						newHead.wear_mask.set_loc(H)
 						newHead.wear_mask = null
+					if (isskeleton(H))
+						var/datum/mutantrace/skeleton/S = H.mutantrace
+						if (S.head_tracker != null)
+							S.head_tracker.linked_human = null
+						newHead.head_type = HEAD_SKELETON
```

Comment:
`head_type` should not be modified since it describes the type of head - not every attached head is a skeleton head.